pyproject.toml

```diff
@@ -71,14 +71,17 @@ skip_glob = ["*examples/*", "*vendored*", "*_vendor*"]
 
 [tool.pytest.ini_options]
 filterwarnings = [
+  # turn warnings from napari into errors, unless explicitly allowed below
+  "error:::napari\\.",
```

Comment:
I'm trying to find a link explaining the `:::` syntax - i.e. why 3 colons but couldn't. I can find the warnings -> error here https://docs.pytest.org/en/stable/warnings.html, but not the bit about making it only from one package. Maybe I just missed it. If you have a link can you respond to this comment with it. No need to add it to the PR, but I'm just curious to understand a bit more about the syntax. Thanks!!

Reply:
Those are warning filter strings from the standard library: https://docs.python.org/3/library/warnings.html#describing-warning-filters  I'll add a link

Reply:
ah love it! thanks for the link!!

Reply:
comment added